README.md

```diff
@@ -1,7 +1,114 @@
-# Tour Savvy App
+# Tour Savvy App Frontend
+<a name="readme-top"></a>
 
+# 📗 Table of Contents
 
-## Authors:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Tour_Savvy_Frontend] <a name="about-project"></a>
+
+Tour_Savvy_frontend is a react application where you can signup/login to a tour booking site, see different packages and make reservations. It uses api backend to handle data and redux to manage state.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="https://react.dev/">React v18.2.0</a></li>
+    <li><a href="https://redux.js.org/">Redux v1.9.7</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap v5.3.2</a></li>
+    <li><a href="https://boxicons.com/">Boxicons v2.1.4</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Packages list and details page**
+- **Reservations list and details page**
+- **Add and Remove Reservation**
+- **Add and Remove Package (Admin)**
+- **Login/Logout and Signup feature**
+- **Responsive and stylish design**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: 
+
+```sh
+ [Nodejs] (https://nodejs.org/en/download/current)
+```
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/Zafron047/tour_savvy_frontend.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd tour_savvy_frontend
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+You also need to set up the api backend using
+
+<a href="https://github.com/zuhebahmed88091/tour_savvy_backend">This repository</a>
+
+### Run tests
+
+To run tests, run the following command: 
+
+Example command:
+
+```sh
+  npm test
+```
```

Comment:
-  [ ] 2 tests are not passing:

![image](https://github.com/Zafron047/tour_savvy_frontend/assets/57421823/dbde26a6-4fc0-4190-93e0-88c86e15fe6e)


Kindly, fix those 2 tests in order to preserve your app's integrity. 👍

Reply:
## @Meltrust 
![jest](https://github.com/Zafron047/tour_savvy_frontend/assets/117600135/7f871be1-b080-4f03-99d3-3a4d256e5c66)


Swiper latest version was conflicting with jest. Which we have now solve by downgrading to stable version 9.
Thanks for the feedback 👍 😄